src/main/java/kotori/Kotori.java

```diff
@@ -0,0 +1,88 @@
+package kotori;
+
+import java.util.Scanner;
+
+import kotori.command.Command;
+import kotori.command.ExitCommand;
+import kotori.command.GreetCommand;
+import kotori.parser.Parser;
+import kotori.storage.Storage;
+import kotori.taskList.TaskList;
+
+
+
+/**
+ * This class represents the bot.
+ * */
+
```

Comment:
Javadoc comments should not leave a line before the class or method it is describing